src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -3,6 +3,7 @@
 import seedu.addressbook.data.person.ReadOnlyPerson;
 
 import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
Is this necessary? It's better to keep commits/updates related to what you're working on, so that the PR is clean and can be easily seen what are the proper changes made.